services/remotecluster/service.go

```diff
@@ -127,6 +145,14 @@ func (rcs *Service) Shutdown() error {
 	return nil
 }
 
+// Active returns true if this instance of the remote cluster service is active.
+// The active instance is responsible for pinging and sending messages to remotes.
+func (rcs *Service) Active() bool {
+	rcs.mux.Lock()
```

Comment:
Is it intentional to share same mutex for methods like `Add/Remove TopicListener`?

Reply:
Yes.  There are low contention (rarely called) APIs and don't need another mutex.